orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/ClientSessionService.java

```diff
@@ -54,7 +54,15 @@ public ClientSession generateClientSession(
             VectorOfTrust effectiveVectorOfTrust,
             String clientName) {
         return new ClientSession(
-                authRequestParams, creationDate, effectiveVectorOfTrust, clientName);
+                authRequestParams, creationDate, List.of(effectiveVectorOfTrust), clientName);
+    }
+
+    public ClientSession generateClientSession(
```

Comment:
I know you didn't add these originally, but what is the point of these methods? Are they for mocking in tests or something?

Reply:
I guess it was an attempt to make code more readable in the case that it would be re-used elsewhere? It is only used in one place so will replace it there and delete this method.

Reply:
Turns out it is used for mocking in `AuthorisationHandlerTest` (ATO-248 needs it) so shall put it back in